lucene/expressions/src/test/org/apache/lucene/expressions/js/TestCustomFunctions.java

```diff
@@ -263,7 +263,7 @@ public void testThrowingException() throws Exception {
     PrintWriter pw = new PrintWriter(sw);
```

Comment:
Perhaps we should import assertj for tests. These assertions are so much cleaner with assertj. Don't know whether hamcrest equivalent exist (maybe it does).

Reply:
Yeah, I added this only for my personal debugging, because this test fails with my change (as the expression is hidden from the stack trace, so the "contains" does not work. But when it fails, you don't see what should be there.

Assertj is really better as it automatically prints and also shortens the printout to only relevant part.

Reply:
I love assertj - it really prints out stuff in more elegant fashion than pure junit+hamcrest.